packages/puppeteer-core/src/common/IsolatedWorld.ts

```diff
@@ -517,12 +520,12 @@ export class IsolatedWorld {
           root,
           timeout,
         },
-        new LazyArg(async () => {
+        new LazyArg(() => {
           try {
             // In case CDP fails.
-            return await this.puppeteerUtil;
+            return this.puppeteerUtil;
```

Comment:
I don't think this can throw without an await here.

Reply:
Obsolete.